src/iop/basecurve.c

```diff
@@ -1792,22 +1792,15 @@ static gboolean _scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer use
 
   if(c->selected < 0) return TRUE;
 
-  int handled = 0;
-  float dy = 0.0f;
-  if(event->direction == GDK_SCROLL_UP)
-  {
-    handled = 1;
-    dy = BASECURVE_DEFAULT_STEP;
-  }
-  if(event->direction == GDK_SCROLL_DOWN)
+  gdouble delta_x, delta_y;
```

Comment:
I added a commit to fix this in the mousewheel scrolling case, while trying when possible to keep the scrolling in the direction of any touch or trackpoint events. As comments note, the latter isn't always possible, so I'm erring on the side of keeping the mousewheel behavior consistent.

It feels right, when possible, for touch scrolling to move the point in the direction of the scrolll...